app/core/Engine.test.js

```diff
@@ -43,22 +43,14 @@ describe('Engine', () => {
   // Use this to keep the unit test initial background state fixture up-to-date
   it('matches initial state fixture', () => {
     const engine = Engine.init({});
-    let backgroundState = engine.datamodel.state;
+    let initialBackgroundState = engine.datamodel.state;
 
-    backgroundState = {
-      ...backgroundState,
-      KeyringController: {
-        ...backgroundState.KeyringController,
-        vault: {
-          cipher: 'mock-cipher',
-          iv: 'mock-iv',
-          lib: 'original',
-        },
-      },
+    initialBackgroundState = {
+      ...initialBackgroundState,
     };
```

Comment:
Nit: We can get rid of this part since we're no longer overwriting any part of this object

```suggestion
    initialBackgroundState = {
      ...initialBackgroundState,
    };
```

Reply:
Thanks for the review! Addressed on this [commit](https://github.com/MetaMask/metamask-mobile/pull/9441/commits/fc7803f14af17455211386b9515c73197cf63b8a)